app/models/user.rb

```diff
@@ -7,7 +9,69 @@ class User < ActiveRecord::Base
   gravtastic default: 'identicon'
 
   # FIXME - DRY up, repeated in Story model
-  JSON_ATTRIBUTES = ["id", "name", "initials", "username", "email"]
+  JSON_ATTRIBUTES = ["id", "name", "initials", "username", "email", "tour", "tour_steps"]
+  TOUR_STEPS = [
```

Comment:
No, we should not mix a domain class (the User) with the tour. You could have a class called `WelcomeTour` inside models and have it there, not all models must be extensions of ActiveRecord 😄